website/src/pages/index.tsx

```diff
@@ -118,9 +120,17 @@ function Home() {
                       <img width={25} src={useBaseUrl('img/firebase-logo.png')} alt="Firebase" />
                     </a>
                   </td>
+                  <td>
+                    {plugin.documentation.length ? (
+                      <a href={plugin.documentation} target="_blank">
```

Comment:
Can we use the Link component to make this relative rather than opening in a new tab?

Reply:
Lets do this in another PR.